CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+Contributing to Mypy
+====================
+
+Welcome!  Mypy is a community project that aims to work for a wide
+swath of Python users and Python codebases.  If you're trying Mypy on
+your Python code, your experience and what you can contribute is
+important to the project's success.
+
+
+Getting started, building, and testing
+--------------------------------------
+
+If you haven't already, take a look at the project's README file and
+the [Mypy documentation](http://mypy.readthedocs.org/en/latest/), and
+try adding type annotations to your file and type-checking it with Mypy.
+
+
+Discussion
+----------
+
+If you've run into behavior in Mypy you don't understand, or you're
+having trouble working out a good way to apply it to your code, or
+you've found a bug or would like a feature it doesn't have, we want to
+hear from you!
+
+Our main forum for discussion is the project's [GitHub issue
+tracker](https://github.com/python/mypy/issues).  This is the right
+place to start a discussion of any of the above or most any other
+topic concerning the project.
+
+We have an IRC channel, `#python-mypy` on irc.freenode.net.  This is
+lightly used, but some Mypy core developers are almost always present;
+feel free to find us there and we're happy to chat.  Substantive
```

Comment:
Hmm, I wouldn't put too much weight on that part of that definition -- I don't think I much see "substantive" used in a way where there was any question that the thing might somehow be fictional or imaginary. OED puts it like this: "That has a firm or solid basis; enduring; important, significant, weighty; of substantial extent or amount, considerable." That much better fits how I'm used to seeing the term used.

"Substantive" also contrasts with "procedural", though references are now telling me that that's a specifically legal usage, so maybe that aspect isn't as helpful as I thought.

What I'm aiming to say here is something like this: we're happy to chat about small questions and non-technical matters, but anything that has actual technical content and is more than a short easy discussion should move to (or start on) the issue tracker. I don't think "substantial" would work as well, but maybe "substantive" is too terse and I should just use more words.